src/components/tuComponent/index.ts

```diff
@@ -32,7 +32,7 @@ export default defineComponent({
 		};
 	},
 	props: {
-		color: { type: String, default: null },
```

Comment:
Why was this removed ?

Reply:
i was testing out and playing around the tucomponent to implement that toggle dark mode so will fix that